applications/GeoMechanicsApplication/tests/cpp_tests/input_utility_stub.cpp

```diff
@@ -0,0 +1,49 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+
+#include "input_utility_stub.h"
+
+namespace Kratos {
+
+InputUtilityStub::InputUtilityStub(const std::string& rParameterJsonString) :
+    mParameterJsonString{rParameterJsonString}
+{
+
+}
+
+Parameters InputUtilityStub::ProjectParametersFrom(const std::string& rProjectFilePath) const
+{
+    Parameters result(mParameterJsonString);
+
+    return result;
```

Comment:
I would change this to the following one-liner:
```suggestion
    return Parameters{mParameterJsonString};
```

Reply:
Agreed, forgot to clean that up after simplification, done!